features/src/autogluon/features/generators/auto_ml_pipeline.py

```diff
@@ -42,6 +42,11 @@ class AutoMLPipelineFeatureGenerator(PipelineFeatureGenerator):
     enable_raw_text_features : bool, default False
         Whether to use the raw text features. The generated raw text features will end up with '_raw_text' suffix.
         For example, 'sentence' --> 'sentence_raw_text'
+    enable_vision_features : bool, default True
```

Comment:
Is this argument actually needed? why would a user specify `image_url` in metadata but not want to leverage the images?

Reply:
possibly for debugging, I agree it isn't really necessary, but doesn't hurt.